servicetest/cgroups/test_cgroups.py

```diff
@@ -0,0 +1,178 @@
+
+# Copyright (C) 2016 - 2017 Pelagicore AB
```

Comment:
For consistency, can we decide how to write this? ping @tobsan (component owner). "2016-2017" or "2016 - 2017"?